### DISCUSSION TITLE
Group Composition (due Thursday 02/02)

### DISCUSSION THREAD

**Discussion by BeaLeydier** [gui2de/ppol768-spring23#116] [Announcements]
Please respond to this thread with 

1. Your group number
2. Your group composition

**Rules**
- Only one person per group responds to this thread
- You cannot pick a group number that has already been picked by someone in this thread
- For the group number that you pick, you need to stick to the group size

**Group number sizes**
- Group 1 : 3 members
- Group 2 : 3 members
- Group 3 : 3 members
- Group 4 : 3 members
- Group 5 : 3 members
- Group 6 : 3 members
- Group 7 : 3 members
- Group 8 : 3 members
- Group 9 : 2 members

If that helps, we have posted a list of enrolled students and their github usernames [at the root](https://github.com/gui2de/ppol768-spring23/blob/71934b2fcae9e89f89666f4def3c23dd39d331d9/Individual%20Assignments/README.md) of the individuals assignments folder.

**Comment by hill-hannah** [comment]
Group #1:
- @pew36 
- @scottabrown99 
- @hill-hannah

**Comment by mikegiunti** [comment]
Group #2:

- @Gustavomurillov 
- @mikegiunti 
- @lgf94

**Comment by elenaspielmann** [comment]
Group #3:
@natespilka - Nate Spilka
@KBGTWN - Keegan Brown
@elenaspielmann - Elena Spielmann

**Comment by XZXinyuZheng** [comment]
Group #4:
@mz547 - Ming Zhou
@sunjw0827 - Jiawei (Selina) Sun
@XZXinyuZheng - Xinyu Zheng

**Comment by darkmaterials** [comment]
Group #5:
@darkmaterials --> Serenity Fan 
@alexacoon --> Alexa Coon 
@bhatiasaloni28 --> Saloni Bhatia

**Comment by Fan411525** [comment]
Group #9:
@Fan411525 - Fan Liu
@yfang133 - Daniel Fang

**Comment by genjuro86** [comment]
Group #6:
@bnhippolyte - BeverlyAnn Hippolyte
@dnr36 - Diana Rivas Garcia
@genjuro86 - Young-Ju Nam

**Comment by marbruno** [comment]
Group # 7
@cmg340 -> Maeve Grady
@emmanyhof  -> Emma Nyhof
@marbruno  -> Marlyn Bruno

**Comment by BeaLeydier** [comment]
Thanks all! Group # 8 will then be @JasmineAdams, @chancebhope, @JLei24
